src/test/java/net/syscon/elite/api/resource/impl/OffenderMovementsResourceImplIntTest.java

```diff
@@ -0,0 +1,135 @@
+package net.syscon.elite.api.resource.impl;
+
+import net.syscon.elite.api.model.ErrorResponse;
+import net.syscon.elite.executablespecification.steps.AuthTokenHelper;
+import org.junit.Test;
+import org.springframework.core.ParameterizedTypeReference;
+import org.springframework.http.HttpMethod;
+
+import java.util.Map;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class OffenderMovementsResourceImplIntTest extends ResourceTest {
+
+    @Test
+    public void schedules_court_hearing() {
+        final var token = authTokenHelper.getToken(AuthTokenHelper.AuthToken.NORMAL_USER);
+
+        final var request = createHttpEntity(token, Map.of(
+                "courtCaseId", -1,
+                "fromPrisonLocation", "LEI",
```

Comment:
Sort on interesting that you need a prison location to create a court hearing, I wonder what happens if they are being held in the court and there are two hearings, would the fromPrisonLocation still be a prison? I guess this models what you have to input to NOMIS

Reply:
Indeed it is really supporting what NOMIS does at this point in time.  Happy to talk about though.